appveyor.yml

```diff
@@ -1,6 +1,6 @@
 os: Visual Studio 2017
 
-version: 1.2.2-alpha-{build}
+version: 2.1.0-alpha-{build}
```

Comment:
Make sense keep this as alpha?

Reply:
I answer this question in another PR, let us talk about it if you need.